metaseq/tasks/streaming_language_modeling.py

```diff
@@ -124,9 +127,12 @@ def __init__(self, args):
         if not has_hf_tokenizers:
             raise ImportError("Please install tokenizers with: pip install tokenizers")
 
-        self.tokenizer = ByteLevelBPETokenizer.from_file(
-            args.vocab_filename, args.merges_filename
-        )
+        if args.hf_tokenizer:
```

Comment:
Is there no way to unify the two somehow?  AKA rewrite the vocab/merges filename into the expected `Tokenizer.from_file(...)` input?

Reply:
Yes, I would like that but I can't afford to do the resulting refactor in other places (e.g. the API) right now.

#308 to mark it.